### DISCUSSION TITLE
How do I set a pub to private?

### DISCUSSION THREAD

**Discussion by wendyxiou** [knowledgefutures/pubpub#1350] [Support]
Hi, I am trying to figure out how to set a pub to private, I kept losing my work when it was a draft so I published it but didn't link to it anywhere on my site so I figured that it would be hard to find but people have found it, not sure how, maybe search engine, and it's not ready to be public so i would like to set the pub back to private somehow without deleting it because i think that will delete my doi which has already been shared and cited. how can I do this and make the pub private again?

**Comment by tefkah** [comment]
(note, not a dev)
I don't think you can set a Pub back to private, but I don't think you need to worry about "losing" your doi, as doi's cannot be deleted (unless very specifically requested). If you make a new pub you can just set the doi to be the doi of the previous pub manually (although I'm not sure if that is a feature everyone can use). You will need to make sure that your new Pub has the same url as the old one though, otherwise the doi won't link to it properly.

**Comment by idreyn** [comment]
Hi Wendy — we don't yet have a way to un-Release a Pub, but you can write to us at help@pubpub.org and we'll sort it out by hand for you. Apologies for the inconvenience!
